core/logger/logger.go

```diff
@@ -63,16 +63,18 @@ func SetLogger(zl *zap.Logger) {
 	logger = &Logger{zl.Sugar()}
 }
 
+const readWritePermission = os.FileMode(0600)
```

Comment:
the common practice from my experience is to declare constants at the top of the file.

Also, the 0600 is still a bit cryptic.  Maybe comment what it does?  something stating it restricts read/write access to the owner